graylog2-server/src/test/java/org/graylog/plugins/map/geoip/MaxmindDataAdapterTest.java

```diff
@@ -206,4 +207,31 @@ public void doGetReturnsResultIfCountryResponseFieldsAreNull() throws Exception
             }
         }
     }
+
+    @RunWith(ConditionalRunner.class)
+    @ResourceExistsCondition(GEO_LITE2_ASN_MMDB)
+    public static class AsnDatabaseTest extends Base {
+        public AsnDatabaseTest() {
+            super(DatabaseType.MAXMIND_ASN);
+        }
+
+        @Test
+        public void doGetSuccessfullyResolvesGooglePublicDNSAddress() {
+            // This test will possibly get flaky when the entry for 8.8.8.8 changes!
+            final LookupResult lookupResult = adapter.doGet("8.8.8.8");
+            assertThat(lookupResult.singleValue()).isEqualTo(15169);
+            assertThat(lookupResult.multiValue())
+                    .extracting("as_number")
+                    .containsExactly(15169);
+            assertThat(lookupResult.multiValue())
+                    .extracting("as_organization")
+                    .containsExactly("Google LLC");
+        }
```

Comment:
Why not use something for testing that is used by MaxMind themself?

https://github.com/maxmind/MaxMind-DB/blob/master/source-data/GeoLite2-ASN-Test.json